bugnote_view_inc.php

```diff
@@ -329,21 +332,17 @@
 				echo string_display_links( $t_activity['note']->note );
 				$t_add_space = true;
 			}
-		} else {
-			if ( !$t_security_token_attachments_delete ) {
-				$t_security_token_attachments_delete = form_security_token( 'bug_file_delete' );
-			}
-
-			print_bug_attachment( $t_activity['attachment'], $t_security_token_attachments_delete );
 		}
 
 		if( isset( $t_activity['attachments'] ) && count( $t_activity['attachments'] ) > 0 ) {
-			if ( !$t_security_token_attachments_delete ) {
```

Comment:
Activities of type _note_ always have an `attachments` key, so the isset() is not necessary

Reply:
I have fixed the code, so it always has attachments array and not a mix of attachment and attachments. This was there because of a bug in `bug_activity_api.php`.